### PR TITLE
Dibels with probes

### PR DESCRIPTION
**Pull request by GabyRangelB**
## Merge Pre-Check
- [x] <kbd>Format</kbd> has been run on all requested files
- [x] `dbt build` successfully run for the entire project[^1]

[^1]: If only requesting 1 file, it's acceptable to run build only for that and its downstream dependencies.

### CODE REVIEW

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql [hunk 1]
```diff
@@ -1,509 +1,830 @@
...
-            bss.assessment_grade,
+    iready_roster as (
+        select *
+        from {{ ref("base_iready__diagnostic_results") }}  -- Lists eligible 3rd and 4th students to take DIBELS based on iReady BOY placement
+        where
+            test_round = 'BOY'
+            and student_grade in ('3', '4')
+            and rn_subj_round = 1
+            and overall_relative_placement_int <= 2
+            and academic_year = '2022-2023'
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:13]: hard-coded year

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql [hunk 2]
```diff
@@ -1,509 +1,830 @@
...
-            u.score,
-            u.level,
-            u.national_norm_percentile,
-            u.semester_growth,
-            u.year_growth,
+    student_enrollment as (
+        select *
+        from {{ ref("base_powerschool__student_enrollments") }}  -- List only active students at or below 4th grade
+        where
+            academic_year = 2022
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:21]: hard-coded year

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql [hunk 3]
```diff
@@ -1,509 +1,830 @@
...
+    ),
 
-            null as score_change,
-            null as probe_number,
-            null as total_number_of_probes,
+    student_schedule as (
+        select *
+        from {{ ref("base_powerschool__course_enrollments") }}  -- List students' active schedule for ELA courses only
+        where
+            cc_academic_year = 2022
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:31]: hard-coded year

**Review by cbini — CHANGES_REQUESTED** [review]

**Comment by cbini** [comment]
remove or re-work the hard-coded years

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql (hunk 2 above)
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:21]: Fixed

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql (hunk 3 above)
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:31]: Fixed

File: src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql (hunk 1 above)
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:13]: academic_year is in that format... i will swap it to academic_year_int

Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:13]: actually no. i cant change it to int... let me left it

Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__dibels_dashboard.sql:13]: i had to do this: left(academic_year,4) = cast({{ var("current_academic_year") }} as string) or i would get a dbt build error for wrong types

**Comment by GabyRangelB** [comment]
@cbini how do I get this back into your radar once I make the code changes? I think I got them all.

**Comment by cbini** [comment]
like you just did, or hit me up in slack

**Review by cbini — APPROVED** [review]

**Pull request merged**